test/cmdlineTests/exp_base_literal/output

```diff
@@ -208,78 +208,157 @@ object "C_81" {
             }
 
             function fun_f_80(var_a_4, var_b_6, var_c_8, var_d_10) -> var__13, var__15, var__17, var__19 {
+                /// @origin exp_base_literal/input.sol:160,164
                 let zero_t_uint256_1 := zero_value_for_split_t_uint256()
                 var__13 := zero_t_uint256_1
+                /// @origin exp_base_literal/input.sol:166,169
                 let zero_t_int256_2 := zero_value_for_split_t_int256()
                 var__15 := zero_t_int256_2
+                /// @origin exp_base_literal/input.sol:171,175
                 let zero_t_uint256_3 := zero_value_for_split_t_uint256()
                 var__17 := zero_t_uint256_3
+                /// @origin exp_base_literal/input.sol:177,181
                 let zero_t_uint256_4 := zero_value_for_split_t_uint256()
                 var__19 := zero_t_uint256_4
 
+                /// @origin exp_base_literal/input.sol:196,200
+                /// @origin exp_base_literal/input.sol:196,197
```

Comment:
In original PR I wanted to remove these multiple notations by using the first one, and I did it that way.
@axic commented out in original PR that we just call this unconditionally, but let's see what should be correct behaviour here